sdcm/nemesis.py

```diff
@@ -1959,6 +1964,19 @@ def check_ssl_reload_log(target_node, file_path, since_time):
 
         self.log.info('hot reloading internode ssl nemesis finished')
 
+    def disrupt_run_unique_sequence(self):
```

Comment:
1) do you want this "unique" as part of all Nemesis (ChaosMonkey for example)?
2) are you aware that it will run on the same target node?

Reply:
> 1. do you want this "unique sequence" as part of all Nemesis (ChaosMonkey for example)?

not sure about this one... @roydahan ?

> 2. are you aware that it will run on the same target node?

yes, this is the idea of this sequence, is to run a predefined sequence of disruptions one after another.

Reply:
`sdcm.nemesis.Nemesis.call_random_disrupt_method` will select this method. Are you sure that you want this to happen?

Reply:
> 2. are you aware that it will run on the same target node?
disrupt_mgmt_repair_cli ==> runs on all nodes.
disrupt_terminate_and_replace_node ==> runs on one node
disrupt_grow_shrink_cluster ==> runs on a new node

Reply:
> `sdcm.nemesis.Nemesis.call_random_disrupt_method` will select this method. Are you sure that you want this to happen?

@roydahan  bentsi is right I would rename this function, and remove the disrupt prefix, or maybe move it to `NemesisSequence` altogether, and even create there multiple sequences and give them names, and that can be controlled from configure

Reply:
1. Yes, I considered adding it to the list of COMPLEX_NEMESIS but I decided to let it work as part of "ChaosMonkey".
The idea is to always test this exact sequence also during longevities (it found several issues when I worked on it).

2. As Yaron said, the target node isn't really relevant here.